pkg/client/mongodb/mongodb.go

```diff
@@ -0,0 +1,48 @@
+package mongodb
+
+import (
+	"context"
+	"github.com/joho/godotenv"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+	"os"
+	"time"
+)
+
+//var DB *mongo.Client = ConnectDB()
+
+var DB, _ = ConnectDB()
+
+func ConnectDB() (*mongo.Client, error) {
+	client, err := mongo.NewClient(options.Client().ApplyURI(os.Getenv("MONGODB_URI")))
+
+	if err != nil {
+		log.Fatal(err)
+	}
+	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
+	err = client.Connect(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	err = client.Ping(ctx, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Print("Connected to MongoDB")
+	return client, nil
+}
+
+func EnvMongoURI() string {
+	err := godotenv.Load()
```

Comment:
The best approach should be either to use viper as a global configuration, or to pass down configuration to this package. Ideally, you would like to minimize dependencies on certain files, especially outside of the main(). Viper is great since you can configure it to use flags, files, different formats etc, and the only dependency you can have to it outside is actually the way you try to pull out the value